model/registry/gen-ai.yaml

```diff
@@ -14,6 +14,34 @@ groups:
               stability: experimental
               value: "openai"
               brief: 'OpenAI'
+            - id: microsoft
```

Comment:
Are we using the company name or the service name? Because maybe this should be `azure`

Reply:
maybe we already have a problem with this attribute, as if I understood it correctly there can be two distinct actors on this - the provider of the model, and the provider where things are running? What are we trying to define here of the two?